src/request.ts

```diff
@@ -180,12 +180,12 @@ export default class Request {
     referrer,
   }: {
     url: string;
-    resourceType: ElectronRequestType;
+    resourceType: string;
```

Comment:
Would it be possible to import the type definition from `electorn` or something like `@types/electron`? Just curious as these typing ensures that all values are handled properly in other places in the project.